bin/templates/project/app/build.gradle

```diff
@@ -30,13 +34,16 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.0'
+        if (cdvHelpers.getConfigPreference('EnableKotlin', 'false').toBoolean()) {
+            classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        }
     }
 }
 
 // Allow plugins to declare Maven dependencies via build-extras.gradle.
 allprojects {
     repositories {
-        mavenCentral();
+        mavenCentral()
```

Comment:
I think that shoud be google() instead mavenCentral()

Reply:
I think this should be proposed in a separate PR (this may be done in PR #507).

Also I think removing semicolons should be proposed in a separate PR.